### PR TITLE
Add doc for the Zendesk event source

### PR DESCRIPTION
**Pull request by antoineco**
[Rendered markdown](https://github.com/antoineco/triggermesh-docs/blob/zd-src/docs/sources/zendesk.md)

### CODE REVIEW

File: docs/sources/zendesk.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+![Bridge overview](../images/zendesk-source/create-bridge-3.png)
+
+A ready status on the main _Bridges_ page indicates that the Zendesk [Target][zd-target] and [Trigger][zd-trigger] were
+successfully created and that the event source is ready to forward events from Zendesk.
+
+![Bridge status](../images/awseventbridge-target/create-bridge-4.png)
+
+To verify the successful deployment of the Zendesk event source, navigate to the [Targets][zd-target] tab of the
+_Extensions_ screen in the Zendesk Admin interface, below the _Settings_ section. The event source instance should have
+created a Target following the naming pattern `io.triggermesh.zendesksource.<triggermesh user>.<source name>`.
```
Comment (JeffNaef-Triggermesh) [inline docs/sources/zendesk.md:39]: io.triggermesh.zendesksource.<namespace>.<source name>.

Comment (JeffNaef-Triggermesh) [reply docs/sources/zendesk.md:39]: ?! oh nooo!! markdown got the best of me lol 
*`io.triggermesh.zendesksource.<triggermesh user>.<source name>.`

Comment (antoineco) [reply docs/sources/zendesk.md:39]: Should users be aware that their user name is a "namespace"? I believe not, this is a Kubernetes notion, not a TriggerMesh notion.

File: docs/sources/zendesk.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Open the Bridge creation screen and add a source of type `Zendesk`.
+
+![Adding a Zendesk source](../images/zendesk-source/create-bridge-1.png)
+
+In the Source creation form, give a name to the event source and add the following information:
+
+* **Email**: Email address associated with the Zendesk account.
+* [**Subdomain**][zd-subdom]: Name of the Zendesk subdomain, without the `zendesk.com` domain or `https://` scheme.
+* [**Token**][zd-token]: API token to communicate with the Zendesk API, as described in the previous section.
```
Comment (sebgoa) [inline docs/sources/zendesk.md:22]: is that the actual token or a reference to the secret containing the token ? If it is a secret than we need to explain it.

File: docs/sources/zendesk.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+The Target is linked to a [Trigger][zd-trigger], which can be found by navigating to the _Triggers_ screen, below the
+_Business rules_ section. This Trigger follows the same naming convention as the matching Target.
+
+![Zendesk Trigger name](../images/zendesk-source/triggers-1.png)
+
+The Trigger defines the condition on which a new event is generated and sent to the Target. In the example below, the
+condition is the creation of a new ticket.
+
+![Zendesk Trigger details](../images/zendesk-source/triggers-2.png)
+
```
Comment (sebgoa) [inline docs/sources/zendesk.md:56]: I am missing some type of conclusion here !!!

File: docs/sources/zendesk.md (hunk 2 above)
Comment (antoineco) [reply docs/sources/zendesk.md:22]: It's a secret dropdown, not a raw value. I'll clarify.

File: docs/sources/zendesk.md (hunk 1 above)
Comment (antoineco) [reply docs/sources/zendesk.md:40]: Done in 6f9fc90

File: docs/sources/zendesk.md (hunk 2 above)
Comment (antoineco) [reply docs/sources/zendesk.md]: Done in 66b3e48

File: docs/sources/zendesk.md (hunk 3 above)
Comment (antoineco) [reply docs/sources/zendesk.md:59]: Done in 6c95596

File: docs/sources/zendesk.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+Open the Bridge creation screen and add a source of type `Zendesk`.
+
+![Adding a Zendesk source](../images/zendesk-source/create-bridge-1.png)
+
+In the Source creation form, give a name to the event source and add the following information:
+
+* **Email**: Email address associated with the Zendesk account.
+* [**Subdomain**][zd-subdom]: Name of the Zendesk subdomain, without the `zendesk.com` domain or `https://` scheme.
+* [**Token**][zd-token]: Reference to a [TriggerMesh secret][tm-secret] containing an API token to communicate with the
```
Comment (JeffNaef-Triggermesh) [inline docs/sources/zendesk.md:22]: Love the added link to the tm secret!

File: docs/sources/zendesk.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+## Verification of external resources
+
+To verify the successful deployment of the Zendesk event source, navigate to the [Targets][zd-target] tab of the
+_Extensions_ screen in the Zendesk Admin interface, below the _Settings_ section. The event source instance should have
+created a Target following the naming pattern `io.triggermesh.zendesksource.<user namespace>.<source name>`.
+
+![Zendesk Target name](../images/zendesk-source/targets-1.png)
+
+The Target is configured to include the configured webhook username and password in each request header.
```
Comment (JeffNaef-Triggermesh) [inline docs/sources/zendesk.md:46]: "The Target is configured to include the configured webhook username and password in each request header."

?*"The Target is configured to include the webhook username and password in each request header."

Comment (antoineco) [reply docs/sources/zendesk.md:46]: Oops, didn't notice that one.

Comment (antoineco) [reply docs/sources/zendesk.md]: Done in [157f525a](https://github.com/triggermesh/docs/compare/e2562255bfd388b37eb4535f3fb3e04efa21683e..157f525a4ea20d69c996b927b0ade05b1fa723ae)

**Pull request merged**